### ISSUE TITLE
Selected tab icon color not white inside toolbar

### ISSUE THREAD

**Issue by mindontheblink** [angular/material#5139]
I noticed that after upgrading to 11.3, the selected tab icon color in a toolbar is no longer white.  I see this in the source code:

```
md-toolbar:not(.md-menu-toolbar) md-icon {
  color: rgb(255, 255, 255);
}
```
which is getting overwritten with 
```
md-tabs .md-tab.md-active, md-tabs .md-tab.md-active md-icon, md-tabs .md-tab.md-focused, md-tabs .md-tab.md-focused md-icon {
    color: rgb(96,125,139);
}
```
I changed this to 
```
md-toolbar:not(.md-menu-toolbar) md-icon {
  color: rgb(255, 255, 255) !important;
}
```
which seems to have resolved my problem.

**Comment by ThomasBurleson** [comment]
@rschmukler - related to theme changes ?

**Comment by rschmukler** [comment]
Fixed by c9f822ddbbee4dc148e62b8007dce5fca1feb0aa
